gpflow/expectations.py

```diff
@@ -171,14 +171,14 @@ def _quadrature_expectation(p, obj1, feature1, obj2, feature2, num_gauss_hermite
 
 # =========================== ANALYTIC EXPECTATIONS ===========================
 
-def expectation(p, obj1, obj2=None, num_gauss_hermite_points=None):
+def expectation(p, obj1, obj2=None, nghp=None):
```

Comment:
:param ...: description
:return: description

follow _reStructuredText_ (reST, sphinx) docstrings 
https://pythonhosted.org/an_example_pypi_project/sphinx.html

Reply:
I have changed the docstrings, do they look ok now?